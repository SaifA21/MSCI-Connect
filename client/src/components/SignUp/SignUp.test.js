import renderer from 'react-test-renderer';
import Email from './EmailField.js';
import Password from './PasswordField.js';


// Unit Test: Saif Abuosba 
describe('Sign Up Component', () => {

    it('renders correctly', () => {
        const tree = renderer
          .create(<Email></Email>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });


    it('renders correctly', () => {
        const tree = renderer
          .create(<Password></Password>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });



})