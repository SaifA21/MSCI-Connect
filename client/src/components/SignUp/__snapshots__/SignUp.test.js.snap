// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign Up Component renders correctly 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root"
  style={
    Object {
      "minWidth": 300,
    }
  }
>
  <label
    className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled Mui-error Mui-error"
    data-shrink={false}
    htmlFor="outlined-basic"
    id="outlined-basic-label"
  >
    Email
  </label>
  <div
    className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline Mui-error Mui-error MuiInputBase-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={true}
      autoFocus={false}
      className="MuiInputBase-input MuiFilledInput-input"
      disabled={false}
      id="outlined-basic"
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;
